src/components/common/connectionDialog/internetConnection.js

```diff
@@ -73,13 +73,6 @@ const InternetConnection = props => {
         }
         message = `Waiting for GoodDollar's server (${servers.join(', ')})`
       }
-      if (isConnectionWeb3 === false) {
```

Comment:
why was this removed?

Reply:
Because after some merge with the master branch this condition duplicates. You can see the same condition a few lines above.